packages/ui/src/components/SearchInput/Key.tsx

```diff
@@ -22,18 +23,43 @@ const Container = styled.div`
   }
 `
 
+export const KEYS_MATCH = {
+  Enter: '↵',
+  ' ': '␣',
```

Comment:
```suggestion
  'Space': '␣',
```

Reply:
@BABAK0T0 the key event is a blank space like I added, I'm not checking the code but the key. Check on https://www.toptal.com/developers/keycode and press you space bar and the `event.key` part